### PR TITLE
Update the outdated comment of RotateCertificates for kubelet

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Now the default value of `RotateCertificates` in kubelet is `true`.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/kubelet/config/v1beta1/types.go [hunk 1]
```diff
@@ -182,7 +182,7 @@ type KubeletConfiguration struct {
 	// Dynamic Kubelet Config (beta): If dynamically updating this field, consider that
 	// disabling it may disrupt the Kubelet's ability to authenticate with the API server
 	// after the current certificate expires.
-	// Default: false
+	// Default: true
```
Comment (liggitt) [inline staging/src/k8s.io/kubelet/config/v1beta1/types.go:185]: this does not default to true

**Comment by liggitt** [comment]
> Now the default value of `RotateCertificates` in kubelet is `true`.

that is not accurate, kubelet config continues defaulting RotateCertificates to false

**Comment by fisherxu** [comment]
> that is not accurate, kubelet config continues defaulting RotateCertificates to false

Okey, get it, I just found the true default value is set in keadm... btw, I guess keadm has other configs so it can set RotateCertificates to true? @liggitt  

https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/componentconfigs/kubelet.go#L194

**Comment by liggitt** [comment]
sure, kubeadm is free to set non-default configuration values

**Pull request closed without merging**